src/main/java/org/timinggame/api/room/repository/impl/PlayerRepositoryImpl.java

```diff
@@ -0,0 +1,14 @@
+package org.timinggame.api.room.repository.impl;
+
+import org.springframework.stereotype.Repository;
+import org.timinggame.api.room.repository.PlayerRepository;
+
+@Repository
+public class PlayerRepositoryImpl implements PlayerRepository {
+
+    @Override
+    public int getPlayerCount(final Long roomId) {
+        // TODO: Redis 연결
+        return 0;
+    }
```

Comment:
```suggestion
    public int getPlayerCount(final Long roomId) {
        throw Exception("TODO: Redis 연결 필요합니다");
    }
```

예시와 유사하게 예외를 던지도록 해놓으면 확실하게 수정할 부분을 찾기 편할 것 같아요

Reply:
그렇게 하면 service 계층하고 테스트 코드도 다 수정해야 되는데, 그게 나을까요?

Reply:
RuntimeException으로 던져버리면 될 것 같아요
테스트에는 영향가지 않을 것 같네요